chirho/explainable/handlers/explanation.py

```diff
@@ -85,13 +91,20 @@ def SearchForExplanation(
     :param alternatives: An optional mapping of names to alternative antecedent interventions.
     :param factors: An optional mapping of names to consequent constraint factors.
     :param preemptions: An optional mapping of names to witness preemption values.
-    :param antecedent_bias: The scalar bias towards not intervening. Must be between -0.5 and 0.5, defaults to 0.0.
-    :param consequent_scale: The scale of the consequent factor functions, defaults to 1e-2.
-    :param witness_bias: The scalar bias towards not preempting. Must be between -0.5 and 0.5, defaults to 0.0.
+    :param antecedent_bias: A scalar bias towards not intervening. Must be between -0.5 and 0.5. Defaults to 0.0.
+    :param consequent_scale: The scale of the consequent factor functions. Defaults to 1e-2.
+    :param witness_bias: A scalar bias towards not preempting. Must be between -0.5 and 0.5. Defaults to 0.0.
     :param prefix: A prefix used for naming additional consequent nodes. Defaults to ``__consequent_``.
+    :param num_samples: The number of samples to be drawn for each antecedent and witness.
+                        Needed if witness and antecedent samples are to be coordinated.
+    :param sampling_dim: The dimension along which the antecedent and witness nodes will be sampled,
+                        to be kept consistent with case sampling.
+
+    :note: If ``num_samples`` is not provided, the antecedent and witness nodes will be sampled independently.
```

Comment:
Isn't it even more important to say what happens when a value for this new parameter is provided?